frontend/src/pages/signup-page.tsx

```diff
@@ -67,7 +67,7 @@ function Signup() {
     <div className="flex-grow cursor-default bg-white py-4 dark:bg-dark-card">
       <div className="m-4 mb-4 flex justify-center">
         <div className="flex w-full items-center justify-center">
-          <h2 className="text-center text-lg font-bold text-black dark:text-dark-primary w-2/4 pl-2 sm:text-xl md:w-3/4 md:pl-48">
+          <h2 className="w-2/4 pl-2 text-center text-lg font-bold text-black dark:text-dark-primary sm:text-xl md:w-3/4 md:pl-48">
             Sign up to WanderLust
```

Comment:
remove `sm:`  wherever you have used

Reply:
i have not changed anything here 
preettier just "w-2/4 pl-2" in front of classname